scripts/update_feedstocks_submodules.py

```diff
@@ -40,8 +42,16 @@
             feedstocks_repo.submodules[feedstock.package_name].remove()
 
         # Add the new submodule.
-        feedstocks_repo.create_submodule(feedstock.package_name, repo_subdir,
-                                         url=feedstock.clone_url)
+        try:
+            feedstocks_repo.create_submodule(feedstock.package_name, repo_subdir,
+                                             url=feedstock.clone_url)
+        except ValueError:
+            warning.warn(
+                    "Unable to add the submodule {}. "
+                    "This is likely because the repo has no commits, "
+                    "which likely means something went wrong with feedstock generation. "
+                    "Will skip adding this submodule and continue.".format(feedstock.package_name)
+            )
```

Comment:
I am not convinced that passing this with just a warning is the way to go but I understand your point. I guess that @pelson is out and he cannot review https://github.com/conda-forge/conda-smithy/pull/106 right today. So go ahead and merge this change. If it turns out to be a bad idea we can just reverse it later.